### PR TITLE
Fixing overflow and layout of paidFor labels

### PR DESCRIPTION
**Pull request by katebee**
## What does this change?

css ph'nglui mglw'nafh Cthulhu R'lyeh wgah'nagl fhtagn 🐙

Besides the css, lets wrap some text in a DIV so it can bully other elements for space, and have class

## What is the value of this and can you measure success?

Desired behaviour:

- Badge label (`Paid for by`) should always be above, and centred over, the badge logo
- Badge logo should never be wider than the card
- Reserve space for the badge label, and badge logo, so that the standfirst text doesn't collide when the label uses the full height
- Not breaking layout anywhere else, in the great game of css-whack-a-mole
- Headache free IE styling?

## BEFORE:

picture 584

picture 585

picture 587

picture 589

## AFTER:

picture 581

picture 586

picture 588

picture 582

## Tested in CODE?

DONE ✅

### CODE REVIEW

File: static/src/stylesheets/module/commercial/_advert.scss [hunk 1]
```diff
@@ -126,9 +126,9 @@
     display: flex;
     align-items: stretch;
 
-    > * {
-        width: 100%;
-    }
+    // > * {
+    //     width: 100%;
+    // }
```
Comment (katebee) [inline static/src/stylesheets/module/commercial/_advert.scss]: This seems to be contributing to layout difficulties on the paidFor containers, removing it hasn't broken anything AFAIK... _yet_

🔥⁉️

Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_advert.scss]: hmm, yeah presumably you are referring to the layout of the badge? If yes, then fair enough. Perhaps have a quick look on IE11 if it has any effect.

**Review by regiskuckaertz — APPROVED** [review]
Woohoo after IE tests (do you have a BrowserStack account? If no, just ask Scott)

**Comment by katebee** [comment]
Ha, good call @regiskuckaertz. 

I ran into another weird height issue in IE11, `badge__links` were messing up the layout, fixed it.

**Comment by regiskuckaertz** [comment]
Smashing 😃  :1st_place_medal:

**Pull request merged**